_posts/blog/2019-07-01-ais-workshop.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "M-Lab Participates in Internet Measurements Workshop at 2019 African Internet Summit"
+author: "Chris Ritzo"
+date: 2019-07-01
+breadcrumb: blog
+categories:
+  - data
+  - data-analysis
+  - community
+---
+
+M-Lab was pleased to be invited to contribute to the Internet Measurements
+workshop at the [2019 African Internet Summit][ais2019], June 15-16, 2019 in Kampala,
+Uganda. M-Lab tech lead, Peter Boothe, and advisor, Georgia Bullen, presented a
+hands-on tutorial on querying and visualiing performance and routing datasets.<!--more-->
+
+[ais2019]: https://www.internetsociety.org/events/africa-internet-summit/2019/
+![AIS 2019 workshop photo 1]({{ site.baseurl }}/images/blog/2019-07-01-ais/ais-1.png)
+
```

Comment:
remove non-staff photos since we're not sure whether consent was asked at this event